docs/source/asking_questions.rst

```diff
@@ -154,3 +160,67 @@ See the parent service's `app configuration <https://github.com/octue/octue-sdk-
 and `app.py file <https://github.com/octue/octue-sdk-python/blob/main/octue/templates/template-child-services/parent_service/app.py>`_
 in the  `child-services app template <https://github.com/octue/octue-sdk-python/tree/main/octue/templates/template-child-services>`_
 to see this in action.
+
+.. _overriding_children:
+
+Overriding a child's children
+-----------------------------
+If the child you're asking a question to has its own children (static children), you can override these by providing the
+IDs of the children you want it to use (dynamic children) to the :mod:`Child.ask <octue.resources.child.Child.ask>`
+method. Questions that would have gone to the static children will instead go to the dynamic children. Note that:
+
+- You must provide the children in the same format as they're provided in the :ref:`app configuration <app_configuration>`
+- If you override one static child, you must override others, too
+- The dynamic children must have the same keys as the static children (so the child knows which service to ask which
+  questions)
+- You should ensure the dynamic children you provide are compatible with and appropriate for questions from the child
+  service
+
```

Comment:
Make an issue to provide a service that allows resolution of children, so instead of sending them directly you can provide a token of some kind to resolve the graph; this will work infinitely deeply.

Here in the docs, make a note that it's an intentional choice to go only one generation deep with this and point to the issue, encouraging people to upvote it or engage us if they require this.